stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -881,15 +881,14 @@ def main(workdir, tree, inputs, options, loop_client):
     paths = mk_paths(workdir, fstype)
     mk_workdirs(paths)
 
-    # Find the directory under `/usr/lib/modules/<kver>` where the
-    # kernel/initrd live. It will be the only entity in there.
-    modules_dir = ensure_glob(os.path.join(deployed_tree, 'usr/lib/modules/*'), n=1)[0]
+    # Find the kernel and initramfs. They should be the only ones on
+    # the system so we'll error here if more than one are found:
+    modules_dir_vmlinuz = ensure_glob(os.path.join(deployed_tree, 'usr/lib/modules/*/vmlinuz'), n=1)[0]
+    modules_dir_initramfs = ensure_glob(os.path.join(deployed_tree, 'usr/lib/modules/*/initramfs.img'), n=1)[0]
```

Comment:
So if I understand correctly, there _will_ be multiple subdirectories (kernel version directories) under `/usr/lib/modules`, but only one will have a kernel and initramfs.

Reply:
Yes. Here we are relaxing the requirement a bit to "just one kernel/initramfs pair" rather than "just one modules dir" because we found a case (and there are probably more) where kmod rpms deliver rpms in other dirs, but it's just the module, not an actual other kernel.

Reply:
> So if I understand correctly, there _will_ be multiple subdirectories

and to be clear.. it is now "there _may_ be multiple subdirectories". When we do our pure RHCOS image builds there will only be one, but if people do a container build and layer a kmod RPM they may have more.

Reply:
Thanks for the clarification!